src/PhpSpreadsheet/Shared/JAMA/Matrix.php

```diff
@@ -742,6 +743,9 @@ public function arrayTimesEquals(...$args)
                         $value = trim($value, '"');
                         $validValues &= StringHelper::convertToNumberIfFraction($value);
                     }
+                    if (is_array($value)) {
+                        $value = Functions::flattenArray($value)[0];
```

Comment:
This could be more robust and changed to:

```php
$flattened = Functions::flattenArray($value);
$firstKey = array_key_first($flattened);
$value = $flattened[$firstKey];
```

Reply:
The whole of the JAMA library should really be replaced completely; the code is awful and without tests, and it's no longer a supported library. Unfortunately, I've yet to find a matrix library that actually uses MS Excel matrix logic, so I've been building my own but that's still Work-in-Progress

Reply:
Would be curious to see your WIP when it's ready.

Can we still progress this fix? I've applied it here but that's not to say it couldn't be caught and rectified higher up in the code. This is just my assessment and what I've found to fix the problem.